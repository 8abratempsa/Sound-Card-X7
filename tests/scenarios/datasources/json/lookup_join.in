octosql "SELECT COUNT(*), SUM(j2.field2), SUM(len(j1.field3)) FROM fixtures/join1.json j1 LOOKUP JOIN fixtures/join2.json j2 ON j1.field1 = j2.field1" -o batch_table
