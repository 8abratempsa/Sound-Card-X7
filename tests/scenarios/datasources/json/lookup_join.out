+-------+------------+-------+
| count | sum_field2 |  sum  |
+-------+------------+-------+
| 18144 |     675864 | 81648 |
+-------+------------+-------+
